Add VRRP tests

SerhiyBoikoPLV opened the pull request:
- test_vrrp_basic_on[bridge]
- test_vrrp_basic_on[port]
- test_vrrp_priority_on[bridge]
- test_vrrp_priority_on[port]
- test_vrrp_basic_down_on[bridge]
- test_vrrp_basic_down_on[port]

Also, updated cleanup functions, since they should not care whether DUTs have any links to TG.

Signed-off-by: Serhiy Boiko <serhiy.boiko@plvision.eu>

taskin0003 reviewed (changes requested):
We should add a little bit of information (comments) about why we are changing the minimum links to zero.

SerhiyBoikoPLV commented:
> We should add a little bit of information (comments) about why we are changing the minimum links to zero.

Added a small comment to the code, updated the commit message to make it more descriptive, and updated the PR description.
Also updated one of the commits because there was a bug in `cleanup_ip_addrs`.

taraschornyiplv commented:
@taskin0003 Can you please review?
